environment-dev.yml

```diff
@@ -2,8 +2,8 @@ name: ultraplot-dev
 channels:
   - conda-forge
 dependencies:
-  - python>=3.9,<3.12
-  - numpy>=2.2
+  - python>=3.9,<3.14
+  - numpy
   - matplotlib>=3.9.3
```

Comment:
```suggestion
  - matplotlib>=3.9
```

Reply:
This would be an issue with basemap having no support for numpy < 2. We need to address that first before we add support higher than 3.12.